ods_ci/tests/Resources/Page/ODH/JupyterHub/Elyra.resource

```diff
@@ -0,0 +1,136 @@
+*** Settings ***
+Documentation    Keywords to interact with Elyra in JupyterLab
+
+
+*** Variables ***
+${SVG_CANVAS} =    //*[name()="svg" and @class="svg-area"]
+${SVG_INTERACTABLE} =    /*[name()="g" and @class="d3-canvas-group"]
+${SVG_PIPELINE_NODES} =    /*[name()="g" and @class="d3-nodes-links-group"]
+${SVG_SINGLE_NODE} =    /*[name()="g" and contains(@class, "d3-draggable")]
+${PROPERTIES_PANEL_BTN} =    //div[@class="toolbar-right-bar"]//button[@class="bx--btn bx--btn--ghost"]
+
+
+*** Keywords ***
+Maybe Migrate Pipeline
+    [Documentation]    Outdated pipelines need to be migrated to the newer Elyra version
+    ${popup} =    Run Keyword And Return Status    Page Should Contain Element    //div[.="Migrate pipeline?"]
+    IF    ${popup}==True
+        Click Element    //button[.="OK"]
+        Save Pipeline Changes
+    END
+
+Set Runtime Image In Pipeline Properties
+    [Documentation]    Sets a default runtime image for the pipeline (currently bugged)
```

Comment:
would the test fails since this kw is bugged?

Reply:
No, because
1) the kw is not bugged
2) the kw is not used
what is bugged is the functionality within Elyra